src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        Random random = new Random();
+        return Colors.values()[random.nextInt(Colors.values().length)].toString();
```

Comment:
let's save the result of `Colors.values()` to the separate local variable `Colors[] colors = Colors.values()` and then use it